src/docs/components/DocsBlock.jsx

```diff
@@ -1,6 +1,7 @@
 // @flow
 import React from 'react';
 import type {Node} from 'react';
+import classnames from 'classnames';
```

Comment:
what do you think about:

```jsx
import cx from 'classnames';
```
and later 
```jsx
className={cx('docs-block', {
  'docs-block--even-columns': evenColumns,
  'docs-block--full-width': fullWidth
})}
```

Reply:
@mpacholec good idea, on my way.

Reply:
solved here and in `Button.jsx`. I guess we could extend this pattern to other components in the future.